src/api/config/environments/development.rb

```diff
@@ -52,6 +52,15 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Things to be done after rails has been fully initialized
+  config.after_initialize do
+    # Sync configuration to the backend if it's running
+    begin
+      ::Configuration.first.write_to_backend
```

Comment:
It works (without `first`) because we refactored this time ago to use `method_missing` and ask to the first item (lines 56 and 57 of models/configuration.rb) :smile: